Source/IntegrationTests/TestFiles/LitTests/LitTest/ast/statement/opaqueBlock.dfy

```diff
@@ -6,14 +6,14 @@ method Foo() returns (x: int)
 {
   x := 1;
   var y := 1;
-  opaque
+  opaque 
```

Comment:
```suggestion
  opaque
```

Reply:
IMO this extra space is not harmful. I'd rather not review a superfluous space in the test files. I'd be in favor of running the Dafny formatter on the test files though, and then it would flag such things, but earlier than the review so they would be cheaper to resolve.